rfc8446.xml

```diff
@@ -2614,7 +2615,6 @@ be sent regardless of whether the Certificate message is empty.</t>
 <t>Structure of this message:</t>
 
 <figure><artwork><![CDATA[
-   /* Managed by IANA */
```

Comment:
This comment was added by me since we did not otherwise mention this enum in the IANA Considerations.  We do now, so it's redundant.